src/llmq/utils.cpp

```diff
@@ -38,6 +38,11 @@ static bool IsV19Active(gsl::not_null<const CBlockIndex*> pindexPrev)
     return DeploymentActiveAfter(pindexPrev, Params().GetConsensus(), Consensus::DEPLOYMENT_V19);
 }
 
+static bool IsV20Active(gsl::not_null<const CBlockIndex*> pindexPrev)
+{
+    return llmq_versionbitscache.State(pindexPrev, Params().GetConsensus(), Consensus::DEPLOYMENT_V20) == ThresholdState::ACTIVE;
```

Comment:
Why not use the new way?

Reply:
that's temporary changes (function is just moved, not changed), they are gone in further commits.

```
 static bool IsV20Active(gsl::not_null<const CBlockIndex*> pindexPrev)
 {
-    return llmq_versionbitscache.State(pindexPrev, Params().GetConsensus(), Consensus::DEPLOYMENT_V20) == ThresholdState::ACTIVE;
+    return DeploymentActiveAfter(pindexPrev, Params().GetConsensus(), Consensus::DEPLOYMENT_V20);
 }
```